spec/controller/user_controller_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+  describe 'GET #index' do
+    it 'returns http success' do
+      get :index
+      expect(response).to be_successful
+    end
+
+    it 'renders the index view' do
+      get :index
+      assert 'title', 'Here is a list of users, as the main page'
```

Comment:
- Good job done so far, but according to the project requirements, you are to write tests that ensure that the correct template is rendered, just as you did in the `user#show` test below. Please fix this 😄